src/indexes.rs

```diff
@@ -224,21 +210,14 @@ impl Index {
     ///    description: String,
     /// }
     ///
-    /// // that trait is used by the sdk when the primary key is needed
-    /// impl Document for Movie {
-    ///    type UIDType = String;
-    ///    fn get_uid(&self) -> &Self::UIDType {
-    ///        &self.name
-    ///    }
-    /// }
     ///
     /// # futures::executor::block_on(async move {
     /// let client = Client::new("http://localhost:7700", "masterKey");
     /// let movies = client.index("get_documents");
     /// # movies.add_or_replace(&[Movie{name:String::from("Interstellar"), description:String::from("Interstellar chronicles the adventures of a group of explorers who make use of a newly discovered wormhole to surpass the limitations on human space travel and conquer the vast distances involved in an interstellar voyage.")}], Some("name")).await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     ///
     /// // retrieve a document (you have to put the document in the index before)
-    /// let interstellar = movies.get_document::<Movie>(String::from("Interstellar")).await.unwrap();
+    /// let interstellar = movies.get_document::<Movie>(&String::from("Interstellar")).await.unwrap();
```

Comment:
```suggestion
    /// let interstellar = movies.get_document::<Movie>("Interstellar").await.unwrap();
```

This should work right?

Reply:
yeah that works